subscription-service/src/main/kotlin/com/egm/stellio/subscription/web/SubscriptionHandler.kt

```diff
@@ -39,6 +40,14 @@ class SubscriptionHandler(
     private val subscriptionEventService: SubscriptionEventService
 ) {
 
+    companion object {
+        @Value("\${pagination.limit.default:}")
+        private const val paginationDefaultLimit: Int = 30
```

Comment:
does the value from the configuration really override the default?

Reply:
also, using a configuration properties class is cleaner. see https://github.com/stellio-hub/stellio-context-broker/blob/feature/load-test-experiments/search-service/src/main/kotlin/com/egm/stellio/search/config/ApplicationProperties.kt for instance.